tensorflow/python/keras/layers/local_test.py

```diff
@@ -288,8 +288,23 @@ def test_locallyconnected_implementation(self, width, data_format):
 
       np.random.seed(1)
       tf_test_util.random_seed.set_seed(1)
-      targets = np.random.randint(0, num_classes, (num_samples,))
-
+      
+      # Following code generates sparse targets and converts them 
+      # to one-hot encoded vectors
+    
+      # Create sparse targets eg. [0,1,2]
+      sparse_targets = np.random.randint(0, num_classes, (num_samples,)) 
+      
+      # Convert to one-hot encoding
+      
```

Comment:
Please remove this blank line too.

Reply:
Made requested changes
Thanks for the support!

Reply:
The tests didn't complete, perhaps due to the latest commit.

Reply:
The `tensorflow-butler` bot removed `ready to pull` label. I hope the PR will get merged 😅